crates/polars-io/src/mmap.rs

```diff
@@ -50,14 +49,13 @@ impl Drop for MMapSemaphore {
     }
 }
 
-/// Open a file to get write access. This will check if the file is currently registered as memory mapped.
-pub fn try_create_file(path: &Path) -> PolarsResult<File> {
+/// Check if the file is currently registered as memory mapped.
+pub fn check_not_mapped(path: &Path) -> PolarsResult<()> {
```

Comment:
I think `ensure_not_mapped` is a slightly better name for this function, as it's essentially an assert but returns a Result. I would expect a "check" function to return a bool.